CMakeLists.txt

```diff
@@ -30,3 +30,14 @@ build_translations(i18n)
 generate_desktop(${CMAKE_SOURCE_DIR} asteroid-settings)
 
 feature_summary(WHAT ALL FATAL_ON_MISSING_REQUIRED_PACKAGES)
+
+set(CPACK_GENERATOR "DEB")
+string(TOLOWER "${CMAKE_PROJECT_NAME}" lcproject_name)
+set(CPACK_DEBIAN_FILE_NAME "${lcproject_name}-${CMAKE_PROJECT_VERSION}.ipk")
+set(CPACK_STRIP_FILES TRUE)
+set(CPACK_DEBIAN_PACKAGE_ARCHITECTURE armv7vehf-neon)
+if (NOT CPACK_PACKAGE_CONTACT) 
+    set(CPACK_PACKAGE_CONTACT bogus@example.org)
```

Comment:
Let's at least seize this chance to make a joke and use dev-null@asteroidos.org

Reply:
To be honest I don't care that much about the joke, especially since we already have asteroid-helloworld that uses bogus@example.org so if it's annoying, don't bother :)